e2e/configs/wdio.sauce.all.browsers.conf.js

```diff
@@ -0,0 +1,90 @@
+const { config } = require('./shared/wdio.sauce.shared.conf');
+
+const screenResolution = '1600x1200';
+const defaultBrowserSauceOptions = {
+  screenResolution,
+  seleniumVersion: '3.141.59',
+  ...(process.env.SAUCE_BUILD_NAME ? { build: process.env.SAUCE_BUILD_NAME } : {}),
+  tunnelIdentifier: process.env.TUNNEL_IDENTIFIER || 'testMe',
```

Comment:
In order for e2e tests to work I had to specify `--tunnel-identifier` as an argument to sauce-connect (`bin/sc -u [username] -k [access-key] -x https://eu-central-1.saucelabs.com/rest/v1 --se-port 4444 --tunnel-identifier testMe`). We should make note of this in the readme.